packages/admin/admin/src/inputWithPopper/InputWithPopper.tsx

```diff
@@ -5,23 +5,73 @@ import * as React from "react";
 
 import { styles } from "./InputWithPopper.styles";
 
-type InputWithPopperComponentsProps = InputBaseProps["componentsProps"] & {
+export type InputWithPopperComponentsProps = InputBaseProps["componentsProps"] & {
     clickAwayListener?: Partial<ClickAwayListenerProps>;
     popper?: Partial<PopperProps>;
     transition?: Partial<TransitionProps>;
     paper?: Partial<PaperProps>;
 };
 
-type InputWithPopperComponents = InputBaseProps["components"] & {
+export type InputWithPopperComponents = InputBaseProps["components"] & {
     Transition?: React.ElementType<TransitionProps>;
 };
 
 type ClosePopper = (focusInput?: boolean) => void;
 
+export type InputWithPopperRenderInputFn = (
+    args: Omit<InputWithPopperProps & WithStyles<typeof styles>, "children" | "renderInput"> & {
+        inputRef: React.RefObject<HTMLElement>;
+        openPopper: () => void;
```

Comment:
Since `Transition` is not needed in `defaultRenderInput`, you could probably remove it here.

```suggestion
    args: Omit<InputWithPopperProps & WithStyles<typeof styles>, "children" | "renderInput" | "components"> & {
        inputRef: React.RefObject<HTMLElement>;
        openPopper: () => void;
        components: Omit<InputWithPopperComponents, "Transition">;
```